Mathlib/Tactic/Core.lean

```diff
@@ -125,18 +132,53 @@ def allGoals (tac : TacticM Unit) : TacticM Unit := do
 def andThenOnSubgoals (tac1 : TacticM Unit)  (tac2 : TacticM Unit) : TacticM Unit :=
   focus do tac1; allGoals tac2
 
-variable [Monad m] [MonadExceptOf Exception m]
+variable [Monad m] [MonadExcept Exception m]
```

Comment:
This change did not break anything but it would benefit from input from an expert.